### ISSUE TITLE
NullReference when implicitly casting null Chat to ChatId

### ISSUE THREAD

**Issue by AskoldMakaruk** [TelegramBots/Telegram.Bot#956]
### Steps to reproduce

```csharp
[Test]
public void Casting()
{
	Chat? chat = null;
	Assert.AreEqual(null, ChatIdMethod(chat));
}

private static ChatId? ChatIdMethod(ChatId? chat)
{
	return chat;
}
```

### Expected  behavior
ChatIdMethod returns null
### Actual behavior
Null reference 
### Screenshots
![зображення](https://user-images.githubusercontent.com/18248541/107121080-34cbf980-6899-11eb-8b76-ad8fd8ac233f.png)

### Environment data

*NuGet Package Version:* 15.7.1

*.NET Version:* net5.0

**Comment by AskoldMakaruk** [comment]
[pr](https://github.com/TelegramBots/Telegram.Bot/pull/957)

**Comment by tuscen** [comment]
Fixed in #983. Now it throws `ArgumentNullException` when trying to cast null to `ChatId`.
